plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/tasks/PostgreDatabaseBackupSettings.java

```diff
@@ -93,7 +94,15 @@ public void setShowViews(boolean showViews) {
         this.showViews = showViews;
     }
 
-    public boolean isUseInserts() {
+	public boolean isCompleteBackup() {
+		return completeBackup;
+	}
```

Comment:
We use 4-space indentation everywhere. Please replace your tabs to 4 spaces everywhere
```suggestion
    public boolean isCompleteBackup() {
        return completeBackup;
    }
```

Reply:
Sorry for the error. 
Do you use any automatic tools to format indentation or to check it?

PS I will fix it as soon as possible

Reply:
We use IDEA with [CheckStyle plugin](https://plugins.jetbrains.com/plugin/1065-checkstyle-idea). Here is our configuration for this plugin https://github.com/dbeaver/dbeaver-common/blob/devel/.github/dbeaver-checkstyle-config.xml
After you install CheckStyle plugin in IDEA, it's possible to use this configuration for IDEA formatter
![image](https://github.com/user-attachments/assets/9793458a-e2c7-49be-b44e-113e57082ec1)

Reply:
Thank you very much for the suggestion. Unfortunately, I don' t have Idea IDE, and I am using Eclipse.
Do you happen to know if there is any tool for Eclipse?
Otherwise, no problem. I am fixing it manually.

Another question for the fix it is correct that i will force push another commit with the fix?

Reply:
Unfortunately, we don't have a common code style schema for Eclipse
I can share mine. It might not match the code style we use in IDEA in some minor aspects, but all major things are fine there
[default_code_style_eclipse.zip](https://github.com/user-attachments/files/18677560/default_code_style_eclipse.zip)

Reply:
the file you provide has produce a lot of change in the code finally i do it manually.
maybe if i figure out a way to extrapolate the eclipse config i will notify you.
Now i change and force push the commit let me know if you notice anything